tests/test_apypie.py

```diff
@@ -1,8 +1,35 @@
+# pylint: disable=invalid-name,protected-access
+import pytest
+
+import apypie
+
+
 def test_init(api):
     assert api
     assert api.apidoc
 
 
+@pytest.mark.parametrize('username,password,expected', [
+    (None, None, None),
+    ('user', None, None),
+    (None, 'pass', None),
+    ('user', 'pass', ('user', 'pass')),
+])
+def test_init_auth(apidoc_cache_dir, username, password, expected):
+    api = apypie.Api(uri='https://api.example.com', apidoc_cache_dir=apidoc_cache_dir.strpath,
+                     username=username, password=password)
+
+    assert api._session.auth == expected
+
+
+def test_init_language(apidoc_cache_dir):
+    api = apypie.Api(uri='https://api.example.com', apidoc_cache_dir=apidoc_cache_dir.strpath,
+                     language='tlh')
+
+    assert api.language == 'tlh'
+    assert api._session.headers['Accept-Language'] == 'tlh'
```

Comment:
should we also assert that `Accept-Language` was also sent in the headers of the actual request?